src/client/components/Logout/Logout.component.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import './Logout.style.css';
+
+const userName = 'Some UserName';
```

Comment:
No hardcoded values in your component. Test data like that belongs in Storybook.

Reply:
Done